modules/admin_manual/pages/installation/system_requirements.adoc

```diff
@@ -31,6 +31,9 @@ For _best performance_, _stability_, _support_ and _full functionality_, we offi
 | Database
 | xref:#database-requirements[MariaDB] 10.5 ^1^
 
+| Redis
+| >= 6
+
```

Comment:
We should probably specify this is only in case connection parameters and SSL/TLS support is needed.

Reply:
If the specify this the audience might raise the question "what is the requirements if the support is not needed?". So what would be our answer to this. We should add this as well upfront.

Reply:
I don't think there are any specific requirements related to the redis server version in case support for connection parameters in combination with SSL/TLS is not required.

Reply:
Then let's simplify it to >=6 in general, even if it only applies to a specific condition. Older versions of Redis aren't supported with security fixes anyway. See https://redis.io/docs/about/releases/.